packages/wdio-sauce-service/src/service.js

```diff
@@ -38,6 +38,7 @@ export default class SauceService {
 
     beforeSuite (suite) {
         this.suiteTitle = suite.title
+        global.browser.execute('sauce:job-name=' + this.suiteTitle)
```

Comment:
We are setting the job name already at the end of the test, no need to make an extra request here

Reply:
yes i know - but in our organization we have many tests running in parallel. a lot of these tests have a higher duration time. people complained about not seeing directly which test cases belongs to which job, so i implemented in our framework to set the test name directly at the beginning.
i thought it would be helpful for other people, too.

Reply:
@pako88 let's make this optional and allow user to opt in for that. The documentation should clearly state that this causes an extra request.

Reply:
agree - i need to look into this deeper to understand it completely - therefore i will split this PR.
in this PR i will remove this line.

Reply:
makes sense

Reply:
ok updated